lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -1613,19 +1615,28 @@ def test_certificate_asset_by_slug(self, mock_get_course_run_details):
                 )
             )
 
+    @skipUnless(name_affirmation_service is not None, 'Requires Name Affirmation')
     @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
-    @ddt.data((True, VerifiedNameStatus.APPROVED),
-              (True, VerifiedNameStatus.DENIED),
-              (False, VerifiedNameStatus.PENDING))
+    @ddt.data((True, 'approved'),
+              (True, 'denied'),
+              (False, 'pending'))
```

Comment:
having to put this in all the tests makes me wonder if the name affirmation service itself should expose these signals so that you could do name_affirmation_service.ApprovedSignal or similar

Reply:
This is a good suggestion, I don't think it's necessary for this change but I'll log another ticket for it